packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```
@@ -203,7 +203,22 @@ var generateLocationId = function (location) {
+    /**
+     * In some cases we see playingEntity?.name that looks like name: { _: 'REAL NAME' } which fails conversion.
+     * We're adding this because name is supposed to be a string ('REAL NAME').
+     * This has something to do with wether "addr" is present in playingEntity when it is not we seem to be hitting this error.
+     * The exact cause is unknown.
+     *
+     * Link to original issue: https://linear.app/metriport/issue/ENG-975/fhir-converter-name-field-not-working
+     */
+    if (
+      typeof location?.playingEntity?.name === "object" &&
+      location.playingEntity?.name &&
+      "_" in location.playingEntity?.name
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)